features/works/work_drafts.feature

```diff
@@ -67,6 +67,7 @@ Feature: Work Drafts
       And the draft "draft to post"
     When I am on drafter's works page
     Then I should see "Drafts (1)"
+      And the page title should include "drafter - Drafts"
```

Comment:
As indicated by the [failing test](https://github.com/otwcode/otwarchive/actions/runs/10926698849/job/30331184518#step:12:108), this is in the wrong spot. It should be one line lower, after the test navigates to the Drafts page.

Reply:
oh thanks, I've been having trouble running cucumber tests locally